File: examples/python/pagination/snapshot.py
from northrays import Northrays


def main():
    northrays = Northrays()

    result = northrays.snapshot.list(page=2, limit=10)
    for snapshot in result.items:
        print(f"{snapshot.name} ({snapshot.image_name})")


if __name__ == "__main__":
    main()

File: examples/ruby/pagination/snapshot.rb
# frozen_string_literal: true

require 'northrays'

northrays = Northrays::Northrays.new

result = northrays.snapshot.list(page: 2, limit: 10)
result.items.each do |snapshot|
  puts "#{snapshot.name} (#{snapshot.image_name})"
end
